ccf_transaction_processor/transaction_processor/enclave_registry.h

```diff
@@ -56,13 +56,13 @@ namespace ccf
   };
   DECLARE_JSON_TYPE(VerificationReport);
   DECLARE_JSON_REQUIRED_FIELDS(VerificationReport,
-  epidPseudonym,
-  id,
-  isvEnclaveQuoteStatus,
-  isvEnclaveQuoteBody,
-  version,
-  nonce,
-  timestamp
+    epidPseudonym,
+    id,
+    isvEnclaveQuoteStatus,
+    isvEnclaveQuoteBody,
+    version,
+    nonce,
+    timestamp
   );
```

Comment:
did you really mean to change the indentation so these don't line up?

Reply:
thank you, the above was intentionally done, to ensure that the particular D/S was intended the same as the others in the same header.